src/api/trackerTrackedEntities.ts

```diff
@@ -179,12 +187,22 @@ export type TrackedOrderField = {
 };
 
 export type TrackedAttributesFields = { type: "trackedEntityAttributeId"; id: Id };
+export type TrackedPager = {
+    page: number;
+    pageSize: number;
+    // Only if requested with totalPages=true
+    pageCount?: boolean;
+    total?: number;
+};
```

Comment:
In the description: "They are also keeping the pagination information outside the object for now, but it will be removed in a future release."

Not sure if we should also take the opportunity to address it here and keep the public type: 

```typescript
export interface TrackedEntitiesGetResponse<Fields> {
    instances: SelectedPick<D2TrackerTrackedEntitySchema, Fields>[];
    pager: TrackedPager;
}
```

And transparently make it work for < 41 (with the same idea than instances). What do you think?